src/components/Hotel/ReviewReservation/index.js

```diff
@@ -1,19 +1,19 @@
 import styled from "styled-components";
 
-export default function reviewHotelCard({ reservation }) {
+export default function reviewHotelCard({ hotelReservationData }) {
   return (
     <Card>
-      <img src={reservation.hotel.image} alt={reservation.hotel.name} />
-      <h1>{reservation.hotel.name}</h1>
+      <img src={hotelReservationData.hotel.image} alt={hotelReservationData.hotel.name} />
+      <h1>{hotelReservationData.hotel.name}</h1>
```

Comment:
Seria uma boa, essas propriedades que tem mais de um nível de profundidade no JSON ter uma validação, algo como: 

`hotelReservationData.hotel?.image`

Só pra garantirmos o front não vai quebrar numa situação que `hotelReservationData.hotel` vir `undefined`
Esse comentário se aplica a situações semelhantes

Reply:
É bom colocar essa validação mesmo quando o componente só renderiza se o hotelReservationData for truthy?

Reply:
Só se for uma garantia que sempre virá o .hotel, nesse caso acredito que não precise